<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_1</class>
 <widget class="QWidget" name="Form_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counters settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Resolution time, us</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>№4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>№2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>№1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>№3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Counters (On/Off)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Form_1</receiver>
   <slot>cnt1(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Form_1</receiver>
   <slot>cnt2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Form_1</receiver>
   <slot>cnt3(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_4</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Form_1</receiver>
   <slot>cnt4(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cnt1(int)</slot>
  <slot>cnt2(int)</slot>
  <slot>cnt3(int)</slot>
  <slot>cnt4(int)</slot>
 </slots>
</ui>
